### PR TITLE
Some formatting of files using a script

### PR DESCRIPTION
**Pull request by aaronfranke**
* POSIX says that text files should end in newline characters. Godot 3.2 will do this automatically for GDScript, but for this PR I ran a script which does this for all text files.

* The script also removed trailing space characters. There were a lot of these.

I only committed changes where I'm sure it's is a good change, such as C/C++ files, markdown files, GDScript files, `plugin.cfg`, and `build.ps1`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ After exhausting these options, please open an issue that describes the error in
 
 ### 2. When should I create function bindings to augment SQLite's set of native functions?
 
-Preferably never.
```
Comment (2shady4u) [inline README.md:116]: These double spaces are necessary for formatting purposes of the README, see [here](https://gist.github.com/shaunlebron/746476e6e7a4d698b373)

File: README.md [hunk 2]
```diff
@@ -92,9 +92,9 @@ Bind a [scalar SQL function](https://www.sqlite.org/appfunc.html) to the databas
 
 ### 1. My query fails and returns syntax errors, what should I do?
 
-There are a couple of things you can do before panicking, namely:
```
Comment (2shady4u) [inline README.md:95]: These double spaces are necessary for formatting purposes of the README, see [here](https://gist.github.com/shaunlebron/746476e6e7a4d698b373)

File: README.md [hunk 3]
```diff
@@ -92,9 +92,9 @@ Bind a [scalar SQL function](https://www.sqlite.org/appfunc.html) to the databas
 
 ### 1. My query fails and returns syntax errors, what should I do?
 
-There are a couple of things you can do before panicking, namely:  
-- Test out if your query is valid by trying it out online at https://sqliteonline.com/.
```
Comment (2shady4u) [inline README.md:96]: These double spaces are necessary for formatting purposes of the README, see [here](https://gist.github.com/shaunlebron/746476e6e7a4d698b373)

File: README.md [hunk 4]
```diff
@@ -92,9 +92,9 @@ Bind a [scalar SQL function](https://www.sqlite.org/appfunc.html) to the databas
 
 ### 1. My query fails and returns syntax errors, what should I do?
 
-There are a couple of things you can do before panicking, namely:  
-- Test out if your query is valid by trying it out online at https://sqliteonline.com/.  
-- Encapsulate all conditional statements in tick-marks ', for example:
```
Comment (2shady4u) [inline README.md:97]: These double spaces are necessary for formatting purposes of the README, see [here](https://gist.github.com/shaunlebron/746476e6e7a4d698b373)

**Comment by aaronfranke** [comment]
* Two spaces at the end of a line in markdown preserves the line break only if there isn't any other reason why the line break should be preserved (in which case they do nothing). For line 116, it already has 2 line breaks, so it's treated as a paragraph. For lines 95-97, those are list items, so they already are treated as multiple lines You can see the formatted document here, it looks correct to me: https://github.com/aaronfranke/godot-sqlite/blob/formatting/README.md

* You should place a license or some other file in the `src/sqlite` folder so that readers know this is third-party code and so that users know where to find the upstream repository. I see that SQLite is public domain, but this could still be stated somewhere. Anyway, I undid those changes.

**Comment by 2shady4u** [comment]
Okay, I added a LICENSE.md for the SQLite source files that should satisfy the requirements.

I also replaced the "2019-present" statement in the original LICENSE.md with "2019-2020", as writing present in a license agreement is actually not permitted.

**Comment by aaronfranke** [comment]
Looks good!

**Pull request merged**